phpstan-baseline.neon

```diff
@@ -5,6 +5,11 @@ parameters:
 			count: 1
 			path: web/modules/custom/dpl_dashboard/src/DplDashboardSettings.php
 
+		-
+			message: "#^Method Drupal\\\\dpl_event\\\\Plugin\\\\Field\\\\FieldFormatter\\\\BranchAddressFormatter\\:\\:viewElements\\(\\) return type has no value type specified in iterable type array\\.$#"
+			count: 1
+			path: web/modules/custom/dpl_event/src/Plugin/Field/FieldFormatter/BranchAddressFormatter.php
+
```

Comment:
Please move this to an `ignoreErrors` entry in `phpstan.neon.dist`.

I this it is perfectly acceptable that you ignore such an error related to Drupals render arrays but ignoring it here will leave the next guy with the same hard-to-fix error when implementing a field formatter.

I think something like this should do the trick:

```
    # Field formatters return render arrays which we cannot provide more detailed typing of.
    - '#Method Drupal\\.*\\Plugin\\Field\\FieldFormatter\\.*\:\:viewElements\(\) return type has no value type specified in iterable type array\.#'
```